### PR TITLE
Update che7 doc referencing the old workspace definition

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

### What does this PR do?
Update che7 doc referencing the old workspace definition.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/15237

### CODE REVIEW

**Review by AndrienkoAleksandr — APPROVED** [review]

File: src/main/pages/che-7/contributor-guide/proc_adding-a-che-theia-plug-in-into-a-workspace.adoc [hunk 1]
```diff
@@ -20,28 +20,31 @@ image::extensibility/adding-plug-ins-in-dashboard.png[link="{imagesdir}/extensib
...
 +
-.Adding plug-ins by configuring a workspace
-image::extensibility/adding-plug-ins-in-workspace-configuration.png[link="{imagesdir}/extensibility/adding-plug-ins-in-workspace-configuration.png"]
+.Adding plug-ins by configuring a component
+image::extensibility/adding-plug-ins-in-workspace-devfile.png[link="{imagesdir}/extensibility/adding-plug-ins-in-workspace-devfile.png"]
+
 +
-Plug-ins are separated by commas (`,`).
-. To add plug-ins, append new records after the separator. A record has the following format:
+Plug-ins are added in YAML format  as array elements in the `components` section.
```
Comment (AndrienkoAleksandr) [inline src/main/pages/che-7/contributor-guide/proc_adding-a-che-theia-plug-in-into-a-workspace.adoc]: Minor, extra space 'format  as'  -> 'format as'

Comment (olexii4) [reply src/main/pages/che-7/contributor-guide/proc_adding-a-che-theia-plug-in-into-a-workspace.adoc]: done

**Review by rkratky — APPROVED** [review]

**Comment by rkratky** [comment]
Thanks, @olexii4. Will you merge this yourself when https://github.com/eclipse/che/issues/15237 is merged?

**Comment by olexii4** [comment]
@rkratky Yes I will merge this

**Pull request merged**